open_academy/security/ir.model.access.csv

```diff
@@ -1,2 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_open_academy_open_academy,open_academy.open_academy,model_open_academy_open_academy,base.group_user,1,1,1,1
+access_open_academy_open_academy,open_academy.open_academy,model_open_academy_course,base.group_user,1,1,1,1
```

Comment:
This file refers to the access given to a certain group of users for certain models, so the standard is to place the name of the module and the model. This for the ID column and name column.

```suggestion
access_open_academy_open_academy_course,open_academy_open_academy_course,model_open_academy_course,base.group_user,1,1,1,1
```